### PR TITLE
[Tool request] maigret #2848

### PR DESCRIPTION
**Pull request by dualfade**

### CODE REVIEW

**Comment by dualfade** [comment]
please review. Will adjust accordingly.

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]
Please put the pkgver function in the PKGBUILD.

File: packages/maigret/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+  cd $pkgname
+  echo $PWD
```
Comment (Edu4rdSHL) [inline packages/maigret/PKGBUILD:26]: Why do you need this `echo` here?

File: packages/maigret/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    cat > "$pkgdir/usr/bin/$pkgname" << EOF
+#!/bin/sh
+cd /usr/share/$pkgname
+source venv/bin/activate
+exec python maigret/maigret.py "\$@"
+EOF
+
+chmod a+x "$pkgdir/usr/bin/$pkgname"
+
```
Comment (Edu4rdSHL) [inline packages/maigret/PKGBUILD:44]: Remove this newline.

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]

File: packages/maigret/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/maigret/PKGBUILD]: removed. My bad. Was testing.

File: packages/maigret/PKGBUILD (hunk 2 above)
Comment (dualfade) [reply packages/maigret/PKGBUILD]: new line removed.

**Comment by Edu4rdSHL** [comment]
Hey @dualfade thank you a lot man for the hard work!

**Pull request merged**